hugo/layouts/get-started/single.html

```diff
@@ -0,0 +1,102 @@
+                            </div>
+                        </div>
+                    </div>
+                    <div class="tab-pane fade" id="startTab03" role="tabpanel" aria-labelledby="startTab03-tab">
+                        <div class="row py-4">
+                            <div class="col-sm-12 col-lg-8 mb-4 mb-md-0">
+                                <h2 class="title-line line-yellow">Executives</h2>
+                                <p>7 out 10 enterprise IT initiatives fail. IT projects fail because executives have no visibility into their software portfolios, Engineering teams and Software Development and maintenance processes. This lack of visibility comes from Software Development Life Cycle data silos and the complexity associated with retrieving, storing and querying all that data at an enterprise scale. At source{d} we're helping IT executives manage their organization at scale through Engineering Effectiveness and Efficiency dashboard. For example, we have helped enterprises improve developer productivity, move to DevOps practices and lead Application Portfolio Rationalization project across teams composed of thousands of developers.</p>
+                                <p>At source{d} we believe we can help enterprises improve every layer of their Engineering Data infrastructure stack. From data extraction, transformation, load (ETL), storage, processing, analytics, access and governance, source{d} provides an end-to-end platform designed to make everyone in your organization an operational data analyst.</p>
+                            </div>
+                            <div class="col-sm-12 col-lg-4 text-center">
+                                <a href="https://go.sourced.tech/engine" class="btn btn-highlight w-100 mb-4">Request a Demo</a>
+                                <a href="news-and-press" class="btn btn-highlight w-100 mb-4">News & Press</a>
+                                <a href="" class="btn btn-purple w-100 mb-4" data-toggle="modal" data-target="#videoModal">1m Video</a>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</section>
+
+{{ partial "footer.html" }}
+
+<div id="videoModal" class="modal fade" tabindex="-1" role="dialog" aria-hidden="true">
+    <div class="modal-dialog modal-lg" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h4 class="modal-title text-dark">1m Video</h4>
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                    <span aria-hidden="true">&times;</span>
+                </button>
+            </div>
+            <div class="modal-body">
+                <div class="embed-responsive embed-responsive-16by9">
+                    <iframe class="embed-responsive-item" src="https://www.youtube.com/embed/Cx1yosOybyQ" allowfullscreen></iframe>
```

Comment:
If the video is playing, and you close the modal, it keeps playing with no way to stop it.
I think we should call `pause` on modal close (I think it's possible using youtube API; otherwise, we should create/delete the video iframe on modal opening/closing), or use any other library to manage modals containing videos.

Reply:
Fixed